portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/ClayAlertToast.path

```diff
@@ -15,6 +15,11 @@
 	<td>//*[@id='ToastAlertContainer']//*[@class='lexicon-icon lexicon-icon-${key_icon}']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>ALERT_TOAST</td>
+	<td>//*[@id='ToastAlertContainer']</td>
+	<td></td>
+</tr>
```

Comment:
Hey @deborasoliveira  Could you use a more specific path?  Any path that already exists could take a look at the `message.path` file